gvr-unittestutils/src/main/java/org/gearvrf/unittestutils/GVRTestUtils.java

```diff
@@ -90,6 +96,16 @@ public GVRTestUtils(GVRTestableActivity testableGVRActivity, OnInitCallback onIn
 
     }
 
+    private static String GetDeviceType() {
+        String TryUrl = GOLDEN_MASTERS_BASE_URL + Build.MODEL;
+        try {
+            return new BufferedReader(new InputStreamReader(new URL(TryUrl).openStream())).readLine().trim();
+        } catch (Exception ex) {
+            Log.e(TAG,"Golden master redirect not found: " + TryUrl);
+            return DEFAULT_DEVICE_TYPE;
```

Comment:
This is how we generate new golden masters. Tests run and fail, new masters are in /sdcard/GearVRFTests